### DISCUSSION TITLE
https://github.com/List-KR/List-KR-Fast 문의

### DISCUSSION THREAD

**Discussion by swbaek81** [List-KR/community#58] [Q&A]
안녕하세요 빠른대응에 감사드립니다

https://github.com/List-KR/List-KR-Fast 관련해서요

현재 ListKR + 몇개 사이트 광고 차단만 추가 해서 시작 하면 어떨까요?

goal.com/kr
https://kr.investing.com
https://www.youtube.com/
https://finance.yahoo.com/

요 정도만 추가해서 시작하고 
업데이트는 한달에 한번 정도만 해주시면 어떨지 건의드려 봅니다

한국인이 몇개의 외국 사이트를 차단하기 위해서 
수만개의 규칙을 가진 adguard base 필터를 사용하는건 불합리 하다고 생각 됩니다 
속도 관점에서 유니콘프로 대비 약점이라고 보여요
애드가드 느려서 안 쓴다는 글이 인터넷에도 많습니다.

의견드려 봅니다 감사합니다.

**Comment by swbaek81** [comment]
https://www.msn.com/ko-kr/feed?ocid=msedgntp&pc=u531
https://www.msn.com/ko-kr

사이트 추가합니다

**Comment by piquark6046** [comment]
월요일에 AdGuard 명의로 filters-list-customizer 를 개발할 수 있는 지 물어보겠습니다.

**Comment by swbaek81** [comment]
어떻게 진행되고 있는지 궁금해서 글 남겨 봅니다

**Reply by piquark6046** [reply]
CTO분이 휴가가셔서 답변이 지연되고 있습니다.

**Comment by swbaek81** [comment]
안녕하세요 진행상황 궁금합니다. 감사합니다.

**Reply by piquark6046** [reply]
Slack 메시지를 다시 남겨놓겠습니다.

**Reply by swbaek81** [reply]
안녕하세요 진행상황 궁금합니다

**Reply by swbaek81** [reply]
이건 포기해야 겠죠? 진행상황이 없네요
